tools/proofers/my_projects.php

```diff
@@ -297,30 +334,213 @@ function sql_order_spec( $order_col, $order_dir )
+        "text_none" => _("No previously proofread or formatted projects are currently available."),
+    ],
+    "active" => [
+        "label" => _("Active"),
+        "text_self" => _("All projects that are not yet posted in which you have proofread or formatted a page."),
+        "text_other" => sprintf(_("All projects that are not yet posted in which %s has proofread or formatted a page."), $username),
+        "text_none" => _("No projects found that have not yet been posted."),
+    ],
+    "posted" => [
+        "label" => _("Posted to PG"),
+        "text_self" => _("All projects that have been posted to Project Gutenberg in which you have proofread or formatted a page."),
+        "text_other" => sprintf(_("All projects that have been posted to Project Gutenberg in which %s has proofread or formatted a page."), $username),
+        "text_none" => _("No projects found that have been posted to Project Gutenberg."),
+    ],
+];
+
+$pool_view_options = [
+    "reserved" => [
+        "label" => _("Reserved"),
+        "text_self" => _("Projects reserved for you to post-process."),
+        "text_other" => sprintf(_("Projects reserved for %s to post-process."), $username),
+        "text_none" => _("No projects reserved for post-processing."),
+    ],
+    "active" => [
+        "label" => _("Active"),
+        "text_self" => _("Projects you checked out for Post-Processing or for Post-Processing Verification. Projects checked out for Post-Processing may be in PP, available for PPV, or in PPV."),
+        "text_other" => sprintf(_("Projects %s checked out for Post-Processing or for Post-Processing Verification. Projects checked out for Post-Processing may be in PP, available for PPV, or in PPV."), $username),
+        "text_none" => _("No projects found."),
+    ],
+    "posted" => [
+        "label" => _("Posted to PG"),
+        "text_self" => _("All projects that are posted to Project Gutenberg for which you are credited as Post-Processor or Post-Processing Verifier."),
+        "text_other" => sprintf(_("All projects that are posted to Project Gutenberg for which %s is credited as Post-Processor or Post-Processing Verifier."), $username),
+        "text_none" => _("No projects found."),
+    ],
+];
+
+    return [$round_view_options, $pool_view_options];
+}
+
+function get_usertext($text_options)
+{
+    global $pguser, $username;
 
-    if ( !isset( $colspecs[$order_col] ) )
+    if($pguser == $username)
+    {
+        return $text_options["text_self"];
+    }
+    else
     {
-        echo "Invalid order_col parameter: '$order_col'. Assuming '$default_order_col'.<br>\n";
-        $order_col = $default_order_col;
+        return $text_options["text_other"];
     }
+}
+
+
+function show_page_menu($all_view_modes, $round_view, $username, $key)
+{
+    global $pguser;
+
+    $qs_username = "";
+    if($pguser != $username)
+        $qs_username = "username=$username&amp;";
+
+    echo "<div id='tabs'>";
```

Comment:
The html checker complains of duplicate ids 'tabs' and 'current'

Reply:
Good catch! I've changed the CSS to be class-based instead of id-based.